ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.11.yaml

```diff
@@ -36,10 +36,10 @@ tests:
 - as: installer-rehearse-aws
   interval: 960h
   steps:
-    cluster_profile: aws-qe
+    cluster_profile: aws-outpost
```

Comment:
I suggest creating separate job configs for AWS Outpost testing instead of cucushift.

Reply:
I don't know how to do that.  what's wrong with current rehearsal?
Anyway the rehearsal is not working!